jdrfandroidbleparser/src/main/java/org/ehealthinnovation/jdrfandroidbleparser/encodedvalue/dis/bgmmeasurement/Flags.kt

```diff
@@ -0,0 +1,31 @@
+package org.ehealthinnovation.jdrfandroidbleparser.bgm.encodedvalue.bgmmeasurement
+
+import java.util.*
+
+/**
+ * These flags define which data fields are present in the Characteristic value
+ * https://www.bluetooth.com/specifications/gatt/viewer?attributeXmlFile=org.bluetooth.characteristic.glucose_measurement.xml
+ */
+enum class Flags constructor(val bit: Int) {
+
+    TIME_OFFSET_PRESENT(1 shl 0),
+    CONCENTRATION_PRESENT(1 shl 1),
+    CONCENTRATION_UNIT(1 shl 2),
+    STATUS_ANNUNCIATION_PRESENT(1 shl 3),
+    CONTEXT_PRESENT(1 shl 4);
```

Comment:
Please use naming similar to spec, GLUCOSE_CONCENTRATION_TYPE_SAMPLE_PRESENT, GLUCOSE_CONCENTRATION_UNITS, SENSOR_STATUS_ANNUNCUATION_PRESENT, etc

Labels are free, we don't charge per keystroke

Reply:
done